pkg/storage/storage.go

```diff
@@ -83,6 +83,7 @@ type ReadAuthorizationModelsOptions struct {
 // ListStoresOptions represents the options that can
 // be used with the ListStores method.
 type ListStoresOptions struct {
+	IDs        *[]string
```

Comment:
Note that @jpadilla has mentioned that he prefers that we don't pass them in the options and instead do a breaking change
```go
type ListStoresFilter {
  IDs *[]string
}

ListStores(ctx context.Context, filter ListStoresFilter, options ListStoresOptions) ([]*openfgav1.Store, []byte, error)
```

I think we can maybe get a PR in as is and fix it once a decision is made

Reply:
I'm fine either way, but please add godoc at the top of `IDs` that explains what this filter is for :)